integration-tests/jfr-opentelemetry/src/main/resources/application.properties

```diff
@@ -1 +1,3 @@
-quarkus.native.monitoring=jfr
\ No newline at end of file
+quarkus.native.monitoring=jfr
+
+quarkus.observability.enabled=false
```

Comment:
Given all this disabling, I think we should have it disabled on tests, by default.

Reply:
Yeah, probably better then all this disabling.